source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -65,9 +151,11 @@ content: |
   ``MongoDBODBC`` DSN:
 
   .. code-block:: shell
-     :copyable: false
 
-     > iusql -v MongoDBODBC
+     iusql -v MongoDBODBC
+
+  .. code-block:: shell
+     :copyable: false
 
      SQL> use test;
```

Comment:
Does `test` work in every circumstance? If not, set up the example in an example block saying something like: "To see what tables are in the test database, invoke these commands:"

Reply:
Explained the example a bit more fully.

Reply:
👍